src/components/Class/styles.scss

```diff
@@ -24,11 +24,15 @@
   .title {
     width: 100%;
     display: flex;
-    justify-content: center;
-    text-transform: uppercase;
+    // justify-content: center;
+    // text-transform: uppercase;
+    margin-left: 21%;
+    font-size: 200%;
```

Comment:
Use px, not percentage. Also, why did we remove justify-content and text-transform? Please also test those components where these styles are being used.

Reply:
I had removed text-transform because in design the title was in small later so I thought we need to keep it like this that's why.